src/pyodide.js

```diff
@@ -329,17 +329,9 @@ globalThis.languagePluginLoader = (async () => {
   ////////////////////////////////////////////////////////////
   // Rearrange namespace for public API
   let PUBLIC_API = [
-    'globals',
-    'loadPackage',
-    'loadPackagesFromImports',
-    'loadedPackages',
-    'pyimport',
-    'runPython',
-    'runPythonAsync',
-    'version',
-    'registerJsModule',
-    'unregisterJsModule',
-    'setInterruptBuffer',
+    'globals', 'loadPackage', 'loadPackagesFromImports', 'loadedPackages',
+    'pyimport', 'runPython', 'runPythonAsync', 'version', 'registerJsModule',
+    'unregisterJsModule', 'setInterruptBuffer', 'pyodide_py'
```

Comment:
This linting change by clang-format doesn't make sense (and as far as I remember it's not the first time). Should we just apply clang format off, and keep one entry per line manually for the `PUBLIC_API`?

Reply:
Yeah I was thinking about that. It seems to be a bit unstable about which it prefers here for some reason.